attributes/volume_attachments_variables.rb

```diff
@@ -0,0 +1,53 @@
+# (c) Copyright 2017 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+# NOTE: variables to be used within server_profile and server_profile_template examples.
+
+# use this to create server profile or server profile template with volume attachments using api200 or api300
+volume_data = {
+  name: 'Volume1',
+  description: 'volume for api200 or api300',
+  provisioningParameters: {
+    provisionType: 'Full',
+    requestedCapacity: 1024 * 1024 * 1024,
+    shareable: false
+  }
+}
+
+# use this to create server profile or server profile template with volume attachments using api500 or greater
+# volume_data = {
+#   name: 'Volume1',
+#   description: 'Volume store serv for api500',
+#   size: 1024 * 1024 * 1024,
+#   provisioningType: 'Thin',
+#   isShareable: false
+# }
+
+# use this to add storage paths to volume attachments using api200 or api300
+storage_paths = [
```

Comment:
I don't like how the examples were split.

Put the data in the same file as the example so the user won't need to alternate between multiple files to see on example.

If needed please create a separate `volume_attachment.rb` example, and if needed create one for API500.

Reply:
I'll split the examples in 4:
- server_profile
- server_profile_template
- server_profile_with_attachments
- server_profile_template_with_attachments

I think that is simples and organized. What do you think?

Reply:
You can have just one example for server_profile_with_attachments and server_profile_template_with_attachments since they'll basically be the same.

So it'll leave you with 3 files:
- server_profile
- server_profile_template
- server_profiles_with_attachments (Or you can simply put these examples inside the previous files, leaving you with two example files)

And, if needed add one more for API500 if the data or behavior differ from the previous ones.

Reply:
ok, the original solution was keeping the volume attachments examples inside the previous files of examples, but the data necessary to run the exemple of attachments made much noises into the files, so we decided move the data to other file. But I think that the solution with 3 files seems good, because the third file will have just eamples about volume attachments, maybe can be named as volume_attachments.rb.
What do you think?